service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -75,10 +73,7 @@ void shouldMapEmptyCaseDataForDraftSDO() throws IOException {
         Map<String, Object> templateData = caseDataExtractionService
             .getStandardOrderDirectionData(CaseData.builder().build());
 
-        assertThat(templateData.get("judgeAndLegalAdvisor")).isEqualToComparingFieldByField(ImmutableMap.of(
-            "judgeTitle", EMPTY_PLACEHOLDER,
-            "legalAdvisorName", EMPTY_PLACEHOLDER
-        ));
+        assertThat(templateData.get("judgeTitleAndName")).isEqualTo(EMPTY_PLACEHOLDER);
```

Comment:
We lost the check for legalAdvisorName, is it intended?

Reply:
for now I will add in assertion that it is empty string - I haven't set legal advisor to be EMPTY_PLACEHOLDER anywhere because it is OK to have no legal advisor. In next story when magistrate is selected this may be handled differently